app/src/main/java/org/dhis2/usescases/datasets/datasetDetail/datasetList/DataSetListAdapter.kt

```diff
@@ -48,16 +54,28 @@ class DataSetListAdapter(
                         viewModel.openDataSet(it)
                     },
                 )
-                ListCard(
-                    listAvatar = card.avatar,
-                    title = ListCardTitleModel(text = card.title),
-                    lastUpdated = card.lastUpdated,
-                    additionalInfoList = card.additionalInfo,
-                    actionButton = card.actionButton,
-                    expandLabelText = card.expandLabelText,
-                    shrinkLabelText = card.shrinkLabelText,
-                    onCardClick = card.onCardCLick,
-                )
+                Column(
+                    modifier = Modifier
+                        .padding(
+                            start = Spacing.Spacing8,
+                            end = Spacing.Spacing8,
+                            bottom = Spacing.Spacing4,
+                        ),
+                ) {
+                    if (position == 0) {
+                        Spacer(modifier = Modifier.size(Spacing.Spacing8))
```

Comment:
Why not add the top padding directly to the list?

Reply:
I tried doing that, however old xml view is still there and the recycler is set up in such a way that when each item is expanded the top border of the first card is hidden. due to the actual viewHolderInflated item, rather than the complete list.  I've opened an issue to rfactor this and remove xml for the next patch [here](https://dhis2.atlassian.net/jira/software/c/projects/ANDROAPP/boards/130?search=remove&selectedIssue=ANDROAPP-6045)